docs/pages/aws-oss-guide.mdx

```diff
@@ -565,9 +560,9 @@ sudo certbot certonly --standalone \
                       --email=$EMAIL
 ```
 
-#### Setup Github Auth
+#### Setup Github auth
```

Comment:
Random Thought, What if we renamed this 'Setup login with Github', or something that makes it clear what Github auth is used for?